material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -1371,16 +1374,31 @@ details.interface > summary::before {
 
 /** Feedback widget styling **/
 .md-feedback {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   width: fit-content;
-  padding: 1em;
-  border: 1px solid var(--mystic);
-  border-radius: 16px;
-  margin: 1rem auto;
+  padding: 0;
```

Comment:
These styles are actually not doing anything. If you go to inspector, you can uncheck the fields to see if they make a difference or not to the rendered element on the page.

This will help you clean up any unnecessary CSS at the end of making changes.

<img width="1788" alt="Screenshot 2025-01-31 at 3 19 15 PM" src="https://github.com/user-attachments/assets/a1b21cd8-608e-4c71-a57a-874553cdd276" />